plugins/available/percol.plugin.bash

```diff
@@ -12,24 +13,21 @@ about-plugin 'Search&Select history with percol'
 # Usage
 ## C-r to search&select from history
 
-_replace_by_history() {
-    if command -v tac>/dev/null; then
-        alias _tac=tac
-    else
-        alias _tac="tail -r"
-    fi
-    local l=$(HISTTIMEFORMAT= history | _tac | sed -e 's/^\ *[0-9]*\ *//' | percol --query "$READLINE_LINE")
-    READLINE_LINE="$l"
-    READLINE_POINT=${#l}
-}
-
+_command_exists percol || return
 
-if command -v percol>/dev/null; then
-    current_version=${BASH_VERSION%%[^0-9]*}
-    if [ $current_version -lt 4 ]; then
-       echo -e "\033[91mWarning: You have to upgrade Bash to Bash v4.x to use the 'percol' plugin.\033[m"
-       echo -e "\033[91m         Your current Bash version is $BASH_VERSION.\033[m"
-    else
-        bind -x '"\C-r": _replace_by_history'
-    fi
+if [[ ${BASH_VERSINFO[0]} -lt 4 ]]; then
+	_log_warning "You have to upgrade Bash to Bash v4.x to use the 'percol' plugin."
+	_log_warning "Your current Bash version is $BASH_VERSION."
+	return
+else
+	bind -x '"\C-r": _replace_by_history'
 fi
```

Comment:
Instead of `else; bind; fi`, could move the `bind` to below the function?
Just feels a tad weird to reference the function before we've introduced it ...